sdk/storage/examples/blob_04.rs

```diff
@@ -30,12 +31,12 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let data = b"1337 azure blob test";
     let mut block_ids = Vec::new();
     for (i, block) in data.chunks(64 * 1024 * 1024 /* 64 MiB */).enumerate() {
-        block_ids.push(i.to_be_bytes());
+        block_ids.push(Bytes::from(format!("{}", i)));
         let hash = md5::compute(block).into();
-        let block_id = (&i.to_be_bytes() as &[u8]).into();
+        let block_id = Bytes::from(format!("{}", i));
 
         let put_block_response = blob
-            .put_block(&block_id, block)
+            .put_block(block_id, block)
```

Comment:
Can `put_block` take a `Into<Bytes>` so you don't have to do the conversion here?

Reply:
Yes I've already done it! 👍